ceph/ceph/templates/bin/_check_mgr.sh.tpl

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+export LC_ALL=C
+
+IS_MGR_AVAIL=`ceph mgr dump | python -c "import json, sys; print json.load(sys.stdin)['available']"`
```

Comment:
in the interests of minimizing the helm footprint, should scripts like this go into ceph-container.git instead?

Reply:
yup, I need to fix it so that it sources entry_point.sh first